src/libs/validationHandler.ts

```diff
@@ -0,0 +1,53 @@
+import { Request, Response, NextFunction } from 'express';
+export default (config) => (req: Request,res: Response,next: NextFunction) => {
+    const errors = [];
+    Object.keys(config).forEach((key) => {
+        const i = 0;
+        const keys = config[key];
+        const locations = keys.in[i];
```

Comment:
To get the field value you should go for first non empty value. For example if for a field `in` field is defined like { ... ,in:["body","query"] ,...} then user should be able to pass input filed in body or query. So you should perform validation on first non empty value.